### ISSUE TITLE
3.25 "Waiting in queue to download more items for you. It will resume downloading in less then 60sec" issue

### ISSUE THREAD

**Issue by Pubs3** [benjaminjackman/looty#67]
As i mentioned at the title i'm getting this "Waiting in queue to download more items for you. It will resume downloading in less then 60sec"

**Comment by prophet03** [comment]
yes it doesnt work, exact reason i made this sir. https://github.com/benjaminjackman/looty/issues/66

**Comment by prophet03** [comment]
stopped working 100% about 2 weeks ago it was working fine, im thinking ggg site blocked the api again or something but no one has responded yet.

**Comment by Pubs3** [comment]
it was working the day before yesterday

**Comment by Pubs3** [comment]
And it is working right now

**Comment by prophet03** [comment]
really? strange mine still isnt working lol can anyone else confirm thiers is working now?
